react-app/client/src/ExcelFunctionality/ExportToCSV.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Button } from '@mui/material';
+import * as FileSaver from "file-saver";
+import * as XLSX from "xlsx";
+
+type Props = {
+  csvData: Object[];
+  fileName: string;
+  chooseRows: Object[];
+};
+
+export const ExportCSV: React.FC<Props> = ({ csvData, fileName, chooseRows }) => {
+  const fileType =
+    "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;charset=UTF-8";
+  const fileExtension = ".xlsx";
```

Comment:
Does this work with legacy Excel file extensions? Do we want to support .xls considering the class has been around for a while? We could leave it as future work but just something to consider.

Reply:
I think we can just support .xlsx since its only exporting the file, I don't think there's other use cases they have that would be broken by xlsx